pulpcore/cli/rpm/remote.py

```diff
@@ -11,29 +11,39 @@
     label_select_option,
     list_command,
     name_option,
+    pulp_option,
     show_command,
     update_command,
 )
 from pulpcore.cli.common.i18n import get_translation
-from pulpcore.cli.rpm.context import PulpRpmRemoteContext
+from pulpcore.cli.rpm.context import PulpRpmRemoteContext, PulpUlnRemoteContext
 
 translation = get_translation(__name__)
 _ = translation.gettext
 
+def _url_callback(ctx: click.Context, param: click.Parameter, value: str) -> str:
```

Comment:
Should this be named `_uln_channel_callback` or `_uln_url_callback`?

Reply:
I would suggest `_uln_url_callback` as the callback is for `url` option.

Reply:
Changed name from `_url_callback` to `_uln_url_callback`.